lib/asn1.js

```diff
@@ -457,14 +457,55 @@ asn1.fromDer = function(bytes, options) {
   var byteCount = bytes.length();
   var value = _fromDer(bytes, bytes.length(), 0, options);
   if(options.parseAllBytes && bytes.length() !== 0) {
-    var error = new Error('Unparsed DER bytes remain after ASN.1 parsing.');
+    var error = new Error('Unparsed bytes remain after ASN.1 parsing.');
     error.byteCount = byteCount;
     error.remaining = bytes.length();
     throw error;
   }
   return value;
 };
 
+/**
+ * Parses an asn1 object from a byte buffer in PEM BER format as DER.
+ *
+ * RFC 7468 specifies PEM data is BER encoded. This call will decode data using
+ * `fromDer` but allow trailing data. This will allow a subset of BER data that
+ * appears in the wild with trailing data. Note that RFC 7468 recommends
+ * to prefer DER over BER encoding as mentioned throughout and described in
+ * Appendix B.
+ *
+ * @reference https://www.rfc-editor.org/rfc/rfc7468
+ *
+ * @param bytes the byte buffer to parse from.
+ * @param [options] object with options or boolean strict flag
+ *          [strict] true to be strict when checking value lengths, false to
+ *            allow truncated values (default: true).
+ *          [parseAllBytes] true to ensure all bytes are parsed
+ *            (default: false)
+ *          [decodeBitStrings] true to attempt to decode the content of
+ *            BIT STRINGs (not OCTET STRINGs) using strict mode. Note that
+ *            without schema support to understand the data context this can
+ *            erroneously decode values that happen to be valid ASN.1. This
+ *            flag will be deprecated or removed as soon as schema support is
+ *            available. (default: true)
+ *
+ * @throws Will throw an error for various malformed input conditions.
+ *
+ * @return the parsed asn1 object.
+ */
+asn1.fromPemBer = function(bytes, options) {
```

Comment:
This seems to break abstractions a little too much for my tastes. I understand the reasoning. I think that adding `asn1.fromBer` would be acceptable -- but we've always tried to avoid implementing an entire BER parser because it's usually not necessary, it's complicated, and a much bigger lift ... so that's not going to happen here.

Since this is PEM-specific, I would recommend that we add this utility only to `pem.js` instead. Perhaps renamed to `pem.toAsn1()` or `pem.berToAsn1()` or similar? Thoughts?